Manuals/User manual.md

```diff
@@ -30,7 +30,7 @@
 
 **The beginning :**
 
-You start your journey in a spaceship on a planet you don’t know about... Working in pair with a scientist, he has a task for you! You will have to explore this planet to understand how it has fallen. Thothan, the scientist, believes the key to the planet’s mystery lies in its ancient ruins, and it’s up to you to uncover the truth. Good luck, explorer. Your journey begins now.
+You start your journey in a spaceship on a planet you don’t know about... Working in pair with a scientist, and it seems he has a task for you! You will have to explore this planet to understand how it has fallen. Thothan, the scientist, believes the key to the planet’s mystery lies in its ancient ruins, and it’s up to you to uncover the truth. Good luck, explorer. Your journey begins now.
```

Comment:
"You're working in pair with a scientist, and it seems like he has a task for you"